syntaxes/pgsql-configuration.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
What's this for?

Reply:
Or was it a rename that git couldn't figure out?

Reply:
This is for configuring auto-complete some brackets, comments etc.

Reply:
Yeah, I figured you just renamed the file. Unfortunately git doesn't track renames...